### ISSUE TITLE
Adjust local server to load files from /build/ not from cdn.../dev/

### ISSUE THREAD

**Issue by JoeGermuska** [NUKnightLab/StoryMapJS#256]
(from #255) when running `fab serve`, Storymap assets should be served from `/build/` not `http://cdn.knightlab.com/libs/storymapjs/dev/`

**Comment by jywsn** [comment]
This is for the entire website?

**Comment by JoeGermuska** [comment]
yes, the local server should always load the local copy of the assets. Is there some case I'm not thinking of where that would be a bad idea?

**Comment by jywsn** [comment]
I can't think of anything off-hand.

**Comment by jywsn** [comment]
Eh... should the "examples" be loaded using the embed page in the build directory, too?  Right now, they are loaded using the "latest" embed page.  In other words, should *everything* be loaded from build or just what is currently loaded from "dev"?

**Comment by jywsn** [comment]
d182288eff7fb1af0f81dbd6d66e664bd45d9a1f
